src/resources/views/logs.blade.php

```diff
@@ -76,23 +76,23 @@
                     delete_button.parentsUntil('tr').parent().remove();
 
                     // Show an alert with the result
-                    new PNotify({
```

Comment:
Actually the syntax is a little _more_ different in V4:

```php
                    // Show an alert with the result
                    new Noty({
                        text: "<strong>{{ trans('backpack::logmanager.delete_confirmation_title') }}</strong><br>{{ trans('backpack::logmanager.delete_confirmation_message') }}",
                        type: "success"
                    }).show();
```

Notice there's no "title", and you need to call the show() method afterwards, in order to actually show it. If you've done this in other places in your app, make sure to make these changes to. I've already added them to your PR before merging.

Reply:
Thank. I'll update them in all places.